### ISSUE TITLE
When gghistostats are made in an R notebook, a blank white page is also made

### ISSUE THREAD

**Issue by ethulin** [IndrajeetPatil/ggstatsplot#27]
When I create gghistostats plot from R notebook, in addition to the plot, I also get this blank white page. Here is a simple chunk that generates the issue.

```{r}
gghistostats(cars,speed)
```

**Comment by IndrajeetPatil** [comment]
I can't seem to reproduce this error.

Configuring the Notebook file:
```
---
title: "R Notebook"
output:
  html_document:
    df_print: paged
  pdf_document: default
---
```

Testing the code:
```{r fig.width=5, message=FALSE}
library(ggstatsplot)
ggstatsplot::gghistostats(data = cars, x = speed, messages = FALSE)
```

I have attached here the HTML file produced. As you can see, there is no extra blank page that I get.

![image](https://user-images.githubusercontent.com/11330453/39712266-6f41891c-51f0-11e8-8177-5e55c75805fb.png)

For the sake of reproducibility, here is my `sessionInfo()`

```
> sessionInfo()
R version 3.4.4 (2018-03-15)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows >= 8 x64 (build 9200)

Matrix products: default

locale:
[1] LC_COLLATE=English_United States.1252  LC_CTYPE=English_United States.1252    LC_MONETARY=English_United States.1252
[4] LC_NUMERIC=C                           LC_TIME=English_United States.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
  [1] nlme_3.1-137           rprojroot_1.3-2        backports_1.1.2        tools_3.4.4            TMB_1.7.13            
  [6] R6_2.2.2               sjlabelled_1.0.10      nortest_1.0-4          lazyeval_0.2.1.9000    colorspace_1.3-2      
 [11] tidyselect_0.2.4       mnormt_1.5-5           emmeans_1.1.3          compiler_3.4.4         expm_0.999-2          
 [16] sandwich_2.4-0         scales_0.5.0.9000      sfsmisc_1.1-2          lmtest_0.9-36          mvtnorm_1.0-7         
 [21] psych_1.8.4            mc2d_0.1-18            ggridges_0.5.0.9000    stringr_1.3.0.9000     digest_0.6.15         
 [26] foreign_0.8-70         minqa_1.2.4            rmarkdown_1.9          stringdist_0.9.4.7     base64enc_0.1-3       
 [31] WRS2_0.9-4             pkgconfig_2.0.1        htmltools_0.3.6        lme4_1.1-17            manipulate_1.0.1      
 [36] pwr_1.2-2              rlang_0.2.0            rstudioapi_0.7.0-9000  jmvcore_0.8.5          shiny_1.0.5           
 [41] bindr_0.1.1            jsonlite_1.5           zoo_1.8-1              dplyr_0.7.5.9000       magrittr_1.5          
 [46] modeltools_0.2-21      bayesplot_1.5.0        Matrix_1.2-14          Rcpp_0.12.16           DescTools_0.99.24     
 [51] munsell_0.4.3          prediction_0.3.2       stringi_1.2.2          multcomp_1.4-8         yaml_2.1.19           
 [56] snakecase_0.9.1        MASS_7.3-50            plyr_1.8.4             grid_3.4.4             parallel_3.4.4        
 [61] promises_1.0.1         ggrepel_0.7.3          sjmisc_2.7.2           forcats_0.3.0          crayon_1.3.4          
 [66] miniUI_0.1.1           lattice_0.20-35        haven_1.1.1.9000       cowplot_0.9.2.9900     splines_3.4.4         
 [71] sjstats_0.14.3         knitr_1.20             pillar_1.2.2           ggpubr_0.1.6           boot_1.3-20           
 [76] rjson_0.2.18           estimability_1.3       reshape2_1.4.3         codetools_0.2-15       stats4_3.4.4          
 [81] glue_1.2.0             evaluate_0.10.1        effsize_0.7.1          data.table_1.11.0      modelr_0.1.1          
 [86] nloptr_1.0.4           httpuv_1.4.2           gtable_0.2.0           purrr_0.2.4.9000       tidyr_0.8.0           
 [91] reshape_0.8.7          assertthat_0.2.0       ggplot2_2.2.1          ggExtra_0.8            mime_0.5              
 [96] coin_1.2-2             xtable_1.8-2           broom_0.4.4            coda_0.19-1            later_0.7.2           
[101] ggcorrplot_0.1.1.9000  rsconnect_0.8.8        survival_2.41-3        tibble_1.4.2           ggstatsplot_0.0.2.9000
[106] glmmTMB_0.2.0          jmv_0.8.6.2            bindrcpp_0.2.2         TH.data_1.0-8   
```

**Comment by ethulin** [comment]
Thanks!

I tried your code exactly. I can replicate that I don't get the blank page when I knit an HTML file. However, my issue (which I should have been clearer about) was when I run the chunk in the workbook itself. I am attaching the screen shot. See here that there is one empty box (which is selected) and then the next box has the graph. When I do the same thing with hist(), I just get the one box with the graph in it.

![image](https://user-images.githubusercontent.com/19850662/39724285-57720fe8-5216-11e8-83df-921cb5c89501.png)

**Comment by IndrajeetPatil** [comment]
@ethulin Thanks for providing more details! I have figured out why this is happening but it will take some time to change this behavior. So, here is a workaround solution for now-

Download the development version:
```
devtools::install_github(
  repo = "IndrajeetPatil/ggstatsplot", 
  dependencies = TRUE,                 
  upgrade_dependencies = TRUE          
)
```
Adjust a new argument I've added and then execute the chunk with the new version:
![image](https://user-images.githubusercontent.com/11330453/39731428-8ef7fdf0-5235-11e8-8863-aaa456e16c6a.png)

As seen here, it should no longer insert a blank image in R Notebook.
